docs/setup.rst

```diff
@@ -69,13 +69,17 @@ run the :doc:`upgrade process <upgrade>` and then proceed to `"Setup" <#setup>`_
 Docker Compose
 ------
 
-1. Make sure you have a Docker machine up and running.
-2. Make sure your current working directory is the root of this GitHub repository.
-3. Run ``docker-compose up postgres``.
-4. Run ``./setup/docker/create_database.sh``. This will access the postgres container and set up the database.
-5. Run ``docker-compose up``
-6. Run ``docker-machine ls``, take note of the ip for the Docker machine you are using, and open the web browser.
-7. Visit that Docker machine IP at port 80, and you should see a Re:dash login screen.
+(``$REDASH_DIR`` refers to the top-level directory of your Redash git repository clone.)
+
+#. Install docker and docker-compose.
+#. ``cd $REDASH_DIR``.
```

Comment:
Why not keep the original instruction (#2 above)? I find it much more understandable than this. You don't use $REDASH_DIR anywhere else anyway.